### DISCUSSION TITLE
Review users and polls

### DISCUSSION THREAD

**Discussion by jcolinger3** [lukevella/rallly#688] [Q&A]
Is there any method to review users on a self hosted?  Using ALLOWED EMAILS and I want to see who has signed up.
Is there a easy method to review all Polls?  Or maybe a simple report of usage?
If in a db, I suppose if we know the structure we could use Redash or something to create a view/report.

**Comment by lukevella** [comment]
There isn't any page like that and it's not something I plan to work on any time soon. My understanding is that the demand for this is relatively low. Your best bet for now is to connect to the database directly which will allow you to see all the registered users and polls that have been created.

**Reply by jcolinger3** [reply]
Thanks for the info Luke. 
I will take a look at the db.
If I create any sql views that are helpful, I'll share.
